tools/run_tests/xds_k8s_test_driver/tests/outlier_detection_test.py

```diff
@@ -55,10 +55,19 @@ def setUpClass(cls):
         https://github.com/grpc/grpc/blob/master/doc/xds-test-descriptions.md#server
         """
         super().setUpClass()
-        if cls.lang_spec.client_lang != _Lang.JAVA:
-            # TODO(mlumish): Once rpc-behavior supported by a language, make the
-            #                override version-conditional.
-            cls.server_image = xds_k8s_flags.SERVER_IMAGE_CANONICAL.value
+        # gRPC Java implemented server "error-code-" rpc-behavior in v1.47.x.
+        if cls.lang_spec.client_lang == _Lang.JAVA:
+            return
+
+        # gRPC CPP implemented server "error-code-" rpc-behavior in v1.55.x.
+        if (cls.lang_spec.client_lang == _Lang.CPP and
+                cls.lang_spec.version_gte('v1.55.x')):
```

Comment:
I just merged C++ changes, they are not getting into 1.56

```suggestion
                cls.lang_spec.version_gte('v1.57.x')):
```